src/render/views/CreateEncounterView.tsx

```diff
@@ -30,26 +46,54 @@ export const CreateEncounterView: React.FC<{}> = () => {
     const viewManager = getViewManager();
     const encounterManager = getEncounterManager();
     const [name, setName] = useObservable("");
+    const [actors, setActors] = useObservableArray<Actor>([]);
+    const [speedDialOpen, setSpeedDialOpen] = useObservable(false);
+    const styles = useStyles();
+    const speedDialActions = [
```

Comment:
You could also define this outside of the component definition as top-level static state so we don't re-initialize a bunch of objects on every render of this component. I do this with most maps and arrays like this one. The performance cost is low but can add up on complex pages.

Reply:
The other option here is to use `const speedDialActions = React.useMemo(() => [ ... ]);` which will memoize the call so it doesn't recalculate it every render. Not super partial here tbh.

Reply:
I actually think I'd prefer `React.useMemo` after thinking on it a bit here. It encapsulates the info we care about inside the component which is clearer and delays the logic until first render.